clients/tabby-chat-panel/src/react.ts

```diff
@@ -1,16 +1,15 @@
 import type { RefObject } from 'react'
 import { useEffect, useRef } from 'react'
 
-import type { ServerApi, ClientApi } from './index'
+import type { ClientApi, ServerApi } from './index'
 import { createClient, createServer } from './index'
 
 function useClient(iframeRef: RefObject<HTMLIFrameElement>, api: ClientApi) {
   const clientRef = useRef<ServerApi | null>(null)
 
   useEffect(() => {
-    if (iframeRef.current && !clientRef.current) {
+    if (iframeRef.current && !clientRef.current)
       clientRef.current = createClient(iframeRef.current, api)
-    }
```

Comment:
Please revert the change. always use `{}` with `if`

Reply:
update the eslint
```
export default antfu({
  rules: {
    curly: ['off', 'all'],
  },
})
```